src/main/cpp/commands/AutoShootCommand.cpp

```diff
@@ -44,21 +41,20 @@ void AutoShootCommand::Execute()
 
 void AutoShootCommand::ShooterWarmup()
 {
-  m_shooterWheel->SetShooter(0.75, 0.75);
+  m_shooterWheel->PIDShoot();
 }
 
 void AutoShootCommand::shoot()
 {
   m_intake->RunIntake(1);
   m_intake->RunMagazine(1);
-  m_intake->DirectionNote(1);
+  m_intake->DirectionNote(1); // why run this?
```

Comment:
We run that one specifically because it runs the center roller to pull up from the back. Depending on the note position in the mag we could remove the RunIntake() and DirectionNote() functions as they might not even be touching the note.